packages/react/src/Button/Button.module.css

```diff
@@ -113,3 +113,17 @@
 .Button--disabled:not(.Button--secondary) {
   box-shadow: none;
 }
+
+@media (forced-colors: active) {
+  /* Retains focus indicator when in "Windows High Contrast Mode". */
+  .Button:focus:not(.Button--disabled[disabled]) {
+    outline: 3px solid;
```

Comment:
FYI, we have some tokens that could perhaps replace the `px` variant?

E.g. 

```css
--brand-borderWidth-thick
--brand-borderWidth-thicker
```

Reply:
Ahh good to know! I replaced this with `--brand-borderWidth-thick` and made note of our tokens.